docs/es/docs/deployment/manually.md

```diff
@@ -65,10 +65,10 @@ And run your application the same way you have done in the tutorials, but withou
 
     </div>
 
-You might want to set up some tooling to make sure it is restarted automatically if it stops.
+Tal vez quedrás configurar algunas herramientas, para asegurarte que se reinicie automáticamente si llega a parar.
```

Comment:
I think `quedrás` is not a valid Spanish word. I would use instead `desee` or `quieras`.